spec/support/helpers/state_file_intake_helper.rb

```diff
@@ -133,6 +141,19 @@ def step_through_df_data_transfer(sample_name = "Transfer my #{filing_year} fede
     click_on I18n.t("general.continue") if expect_success
   end
 
+  def page_change_check(text, sleep_time: 0.1)
+    begin
+      expect(page).to have_text(text)
+    rescue Selenium::WebDriver::Error::WebDriverError,
+      Capybara::ElementNotFound,
+      RSpec::Expectations::ExpectationNotMetError => e
```

Comment:
I'm only seeing `RSpec::Expectations::ExpectationNotMetError` now in my logs when I watch the tests run -- are the other two still necessary?

Reply:
Yes sometimes locally I would get the other two, they would be less common though